_posts/2020-11-30-homemade-oreos.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Homemade Oreos"
+date:   2020-11-29
```

Comment:
Can you change this to 11/30?

Reply:
Wait, you expect me to *read* and *think* about what I'm copying??? How rude.

(fixed)

Reply:
I would like a recipe for copypasta on my desk by end of week 😛

Reply:
You failed to reckon with the fact that I've been on vacation for a week and don't want to actually get back to work. 😄  https://github.com/aloria/aloria.github.io/pull/6